app/src/main/java/org/sagebase/crf/step/active/HeartBeatUtil.java

```diff
@@ -29,26 +29,57 @@ public static HeartBeatSample getHeartBeatSample(double timestamp, Bitmap bitmap
         HeartBeatSample sample = new HeartBeatSample();
         sample.t = timestamp;
 
+        long redCount = 0;
         long r = 0, g = 0, b = 0;
+        int height = bitmap.getHeight();
+        int width = bitmap.getWidth();
         int intArray[] = new int[bitmap.getWidth() * bitmap.getHeight()];
```

Comment:
Nit: replace getWidth/getHeight with the `height` and `width` local vars you added

Reply:
Oops. Might have mashed the wrong button. Meant to approve, not request changes